src/components/ListView.js

```diff
@@ -38,6 +40,7 @@ class ListView extends Component {
         <div className="cardContainer">
           {reports.filter(report => dataMatchesFilter(report, filter))
             .map((report) => <ListCard report={report} key={report.id}/>)}
+            <List onClick={() => history.push('/')} className="listMapToggle"/>
```

Comment:
Can this be a map icon instead? It's a little confusing to click on a list button from the list view and end up on a map view